### ISSUE TITLE
Getting very low map during mosaic training in 2 channel training

### ISSUE THREAD

**Issue by ajithvallabai** [ultralytics/ultralytics#2162]
### Discussed in https://github.com/orgs/ultralytics/discussions/2138

<div type='discussions-op-text'>

<sup>Originally posted by **ajithvallabai** April 20, 2023</sup>
Hi, 
 I am trying to train depth images (two depth images of single channel are merged and being fed to the training). Each of the depth image contains values from 0 to 2000+ and after training i am able to predict properly also. Lets take a example of 40 epochs by default 30 epochs will be mosaic training and remaining 10 epochs will be normal training, during the first 30 epochs the map is only around 0.00xxx to 0.0xxxx but in the last 10 epochs it is raising from 0.5x to 0.72 map i am not sure why its happening like this. i have attached sample logs for 20 epochs where first 10 is mosaic and second 10 is normal . I tried to adjust normalization values img/=255 to img/=2000 but got similar results . kindly help me with this
version 8.0.24
[snap_logs.txt](https://github.com/ultralytics/ultralytics/files/11283869/snap_logs.txt)
</div>

**Comment by ajithvallabai** [comment]
problem was in below line in augument.py file 

`img4 = np.full((s * 2, s * 2, img.shape[2]), 114, dtype=np.uint8)`

since it was uint8 values were being clipped to 0 to 255 range when i changed to np.int16 it was working fine in mosaic mode . Any other suggestions to make 2 channel training better kindly let me know

**Comment by pderrenger** [comment]
@ajithvallabai glad to hear that you resolved the issue by changing the data type to np.int16! 👍 When working with 2 channel training, you may consider experimenting with different data augmentations, regularization techniques, and hyperparameters to further enhance the performance. Additionally, adjusting the normalization process specific to your depth images might also have an impact. If you encounter any other challenges or have further questions, feel free to ask for more assistance. Happy training!

**Comment by glenn-jocher** [comment]
@pderrenger It's great to see you've identified the data type issue and are making progress with your 2-channel training! 🎉 Your approach to experimentation is key. Keep fine-tuning those augmentations and hyperparameters to optimize your model's performance. If you hit any more snags or have questions, we're here to help. Keep up the good work!
